### ISSUE TITLE
Fresh YEdit with Eclipse dark theme : value as black on a dark grey background

### ISSUE THREAD

**Issue by nicolas-albert** [oyse/yedit#46]
I have just installed YEdit on my Eclipse Neon configured with the dark theme.
I think, default foreground color for this theme should be review, especially the value (black).
Also I didn't find how to change this color in the Eclipse preferences page.

Do you have a workaround ?

Thx.

**Comment by oyse** [comment]
Not really. I think the easiest solution is to allow setting the default foreground color as well. Feel free to provide a pull request if you have the time.

I looked into providing support for switching colors when the theme was switched, but this was surprisingly difficult actually.

**Comment by nicolas-albert** [comment]
Sorry, I don't have many time too make a custom build for changing a background color, so I use simple Text Editor when I need more contrast.

**Comment by lifeofguenter** [comment]
+1

**Comment by kbcmdba** [comment]
Another +1... One of the bigger features of Eclipse Oxygen was that it would have a much better-looking dark theme.

**Comment by danfournier** [comment]
+1

**Comment by nicolas-albert** [comment]
Still no workaround (or a fix) ?

**Comment by mabazyar** [comment]
Any luck?

**Comment by manatlan** [comment]
sad ;-(

**Comment by guisty** [comment]
You can go to _**Window -> Preferences -> YEdit Preferences -> Color preferences**_ so you can change the text color in editor

**Comment by kcbaltz** [comment]
YEdit doesn't seem to have "Color Preferences" anymore.  
![image](https://user-images.githubusercontent.com/8388083/119718905-01e44c00-be1d-11eb-81e0-ae5128574cb1.png)

**Comment by pmahony893** [comment]
> YEdit doesn't seem to have "Color Preferences" anymore.
> ![image](https://user-images.githubusercontent.com/8388083/119718905-01e44c00-be1d-11eb-81e0-ae5128574cb1.png)

It is there, but is filtered out in your screenshot by your search term "yedit". If you search instead for **Color Preferences** (or indeed just scroll down without using the search), you will see it.

![image](https://user-images.githubusercontent.com/32841410/121372373-26b8e300-c936-11eb-9aec-42e5512416cf.png)
